lib/src/polymer_base.dart

```diff
@@ -229,6 +229,12 @@ abstract class PolymerBase implements CustomElementProxyMixin {
         .apply([path, convertToJs(value), fromAbove], thisArg: jsElement);
   }
 
+  /// Notify that a value at a path has been changed (without passing the value).
+  void notifyPath1(String path) {
```

Comment:
Hmm, I didn't realize there was already a named parameter to notifyPath, that is unfortunate. I really don't want to go down the path of adding a number to the end of the method name just to overload it, so lets try to figure out something else.

I don't think that the `fromAbove` named parameter is actually used by much user code, so we could possibly change it to match the JS signature and just use two optional parameters? That would likely break polymer.dart itself (although it would be an easy fix), but not much user code.